### DISCUSSION TITLE
ssd speed for tablebase

### DISCUSSION THREAD

**Discussion by zecke345** [official-stockfish/Stockfish#4299] [Q&A] [answered]
hi

i wanna know if there is a noticeable strength difference putting 6-men tablebases on different speed SSD's.
For ex is SF much stronger when tablebases are on a normal Sata SSD, m.2 with 3k gb reading, and highend m.2 with 7k reading speed.
is there a difference, if any?

should the tablebases be on a different ssd than the operating system

ty

**Comment by vondele** [comment, marked as answer]
So, even putting TB in a RAM disk, there is really little Elo to be gained:
https://github.com/official-stockfish/Stockfish/wiki/Useful-data#tb6-testing-for-various-versions-of-sf

With a slower SSD you might be losing Elo... but I have no explicit numbers.

**Comment by zecke345** [comment]
so there is basically no difference if u use a tablebase or not? because with 6man its only 3 elo more?

**Reply by vondele** [reply]
correct. There might be a difference when studying endgames specifically, but the benefit is surprisingly small. SF has gotten really good at endgames.
